db/00000000000010_rm_client_report_fk.down.sql

```diff
@@ -0,0 +1,5 @@
+ALTER TABLE report_aggregations DROP COLUMN client_timestamp;
+ALTER TABLE report_aggregations DROP COLUMN client_report_id;
+ALTER TABLE report_aggregations ADD COLUMN client_report_id BIGINT NOT NULL;
+ALTER TABLE report_aggregations ADD CONSTRAINT fk_client_report_id FOREIGN KEY(client_report_id) REFERENCES client_reports(id) ON DELETE CASCADE;
+CREATE INDEX report_aggregations_client_report_id_index ON report_aggregations(client_report_id);
```

Comment:
Need to delete index rather than create.

(I think this should have failed the CI though... isn't creation of a duplicate index an error?)

Reply:
It is an error...
```
postgres@127:postgres> create table foo(id int primary key, bar int);
CREATE TABLE
Time: 0.014s
postgres@127:postgres> create index baz on foo(bar);
CREATE INDEX
Time: 0.013s
postgres@127:postgres> create index baz on foo(bar);
relation "baz" already exists
Time: 0.002s
```
That suggests the smoke test from https://github.com/divviup/janus/pull/1434 is not working. I will investigate.

Reply:
Nevermind! I see the column is dropped and the index has to be recreated.

Reply:
Deleting the `client_report_id` column [as part of a delete/recreate operation] also deletes any indexes on the column, such as `report_aggregations_client_report_id_index`. Surprisingly, since both the down & up migrations do a delete/recreate column operation, both the down & up migration need to recreate the index.